packages/react-vapor/src/components/form/FormGroup.tsx

```diff
@@ -0,0 +1,17 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+
+export interface IFormGroupProps {
+    title: string;
+    description?: string;
+    className?: string | string[];
```

Comment:
```suggestion
    className?: string;
```
I think it's better to stay closer to react types